Add possibility to init with custom host (#14)
contentful-userland/contentful-wizard: README.md

> There is no "smart" guessing strategy (like trying to list all fields with string value and get one with a short enough value), since it can easily introduce incosistency, and it will become confusing.

## Initialization

> In default configuration there is pre-configured host for you published entries and assets.

> > If you need work with preview items or custom host you have to set in you init script following.

Reviewer: ```suggestion
> If you need to work with preview items define a custom host as `init` parameter.
```

```js
CTFLWizard.init({
  spaceId: 's25qxvg',
  key: 'f78aw812mlswwasw', // your API key

>   host: 'preview.contentful.com' //Preview host

Reviewer: ```suggestion
  host: 'preview.contentful.com' // preview host
```
